WebRoot/js/zimbraMail/package/Startup1_1.js

```diff
@@ -120,6 +120,7 @@ AjxPackage.require("ajax.dwt.widgets.DwtMessageDialog");
 AjxPackage.require("ajax.dwt.widgets.DwtOptionDialog");
 AjxPackage.require("ajax.dwt.widgets.DwtChooser");
 AjxPackage.require("ajax.dwt.widgets.DwtTimeSelect");
+AjxPackage.require("ajax.dwt.widgets.DwtSelect");
```

Comment:
we should remove this package from Startup2.js package

Reply:
Done